aws/distributionscanner.go

```diff
@@ -12,13 +12,15 @@ import (
 	"github.com/quay/claircore/indexer"
 )
 
-// AWS Linux keeps a consistent os-release file between
+// Amazon Linux keeps a consistent os-release file between
 // major releases.
 // All tested images on docker hub contained os-release file
+// scannerVersion matches the most recent major version of
+// Amazon Linux that we have tested with.
 
 const (
 	scannerName    = "aws"
-	scannerVersion = "v0.0.1"
+	scannerVersion = "v2023.0.0"
```

Comment:
We've gone down the route of very simple scanner versions (i.e. "2"), it's used to inform the index_state and semver (or the like) is really overkill

Reply:
I'll replace with "2" then.